### PR TITLE
rush change should respect version policy

### PR DESCRIPTION
**Pull request by qz2017**

### CODE REVIEW

File: apps/rush-lib/src/cli/actions/ChangeAction.ts [hunk 1]
```diff
@@ -237,16 +243,35 @@ export default class ChangeAction extends BaseRushAction {
...
-          } as IChangeInfo;
         }
       });
   }
 
+  private _getBumpOptions(packageName: string): {[type: string]: string } {
+    const project: RushConfigurationProject | undefined = this.rushConfiguration.getProjectByName(packageName);
+    const versionPolicy: VersionPolicy | undefined = project!.versionPolicy;
+
+    let bumpOptions: { [type: string]: string } = {
```
Comment (pgonzal) [inline apps/rush-lib/src/cli/actions/ChangeAction.ts]: >bump [](start = 8, length = 4)

"bump" is not professional terminology.

Could we call it something like "increment" or "upgrade"?

Comment (qz2017) [reply apps/rush-lib/src/cli/actions/ChangeAction.ts]: It is used everywhere in semver.

File: apps/rush-lib/src/cli/actions/ChangeAction.ts [hunk 2]
```diff
@@ -237,16 +243,35 @@ export default class ChangeAction extends BaseRushAction {
...
+      'major': 'major - for breaking changes (ex: renaming a file)',
```
Comment (pgonzal) [inline apps/rush-lib/src/cli/actions/ChangeAction.ts]: >(ex: renaming a file) [](start = 45, length = 21)

Renaming a file is not a good example.  How about:

```
major -- for changes that break compatibility, e.g. removing an API
minor -- for backwards compatible changes, e.g. adding a new API
patch -- for changes that don't affect compatibility, e.g. fixing a bug
```

File: apps/rush-lib/src/cli/actions/ChangeAction.ts (hunk 1 above)
Comment (qz2017) [reply apps/rush-lib/src/cli/actions/ChangeAction.ts]: It is used everywhere in semver.

---
In reply to: [147508119](https://github.com/Microsoft/web-build-tools/pull/416#discussion_r147508119) [](ancestors = 147508119)

Comment (pgonzal) [reply apps/rush-lib/src/cli/actions/ChangeAction.ts]: Example?

---
In reply to: [147509693](https://github.com/Microsoft/web-build-tools/pull/416#discussion_r147509693) [](ancestors = 147509693,147508119)

Comment (qz2017) [reply apps/rush-lib/src/cli/actions/ChangeAction.ts]: [http://semver.org/](http://semver.org/) 

---
In reply to: [147509852](https://github.com/Microsoft/web-build-tools/pull/416#discussion_r147509852) [](ancestors = 147509852,147509693,147508119)

Comment (pgonzal) [reply apps/rush-lib/src/cli/actions/ChangeAction.ts]: I only see one mention of it in a FAQ answer

---
In reply to: [147510469](https://github.com/Microsoft/web-build-tools/pull/416#discussion_r147510469) [](ancestors = 147510469,147509852,147509693,147508119)

Comment (qz2017) [reply apps/rush-lib/src/cli/actions/ChangeAction.ts]: [https://docs.npmjs.com/cli/version](https://docs.npmjs.com/cli/version) 

---
In reply to: [147510748](https://github.com/Microsoft/web-build-tools/pull/416#discussion_r147510748) [](ancestors = 147510748,147510469,147509852,147509693,147508119)

**Pull request merged**